packages/core/src/events/method-events/method-completed-event.class.ts

```diff
@@ -0,0 +1,16 @@
+import { Probe } from '../../probes';
+import { AngularClassType } from '../../probes/module-methods-probe';
+import { MethodEventSupport } from './method-event-support.class';
+
+export class MethodCompletedEvent extends MethodEventSupport {
+  constructor(
+    probe: Probe,
+    classType: AngularClassType,
+    clazz: any,
```

Comment:
This could be misleading. What is the intention of this class. Perhaps using the previously mentioned type alias to indicate intention would help (or use jsdoc style comments for the ctor.

Reply:
I'm not even sure if this probe is even needed actually andrew... i think we can actually delete it? I don't know why we have it honestly.